Example/UserAccountsView.swift

```diff
@@ -44,86 +27,45 @@ extension Font {
 /**
 	Main view of User Accounts preference screen
  */
-struct UserAccountsView: View {
+struct UserAccountsView: View, PreferencePaneView {
 	@State private var isOn1 = true
 	@State private var isOn2 = false
 	@State private var selection1 = 1
 	@State private var selection2 = 0
-	let width: CGFloat = 510.0
+	let contentWidth: CGFloat = 450.0
+	let preferencePaneIdentifier: PreferencePaneIdentifier = .userAccounts
+	let preferencePaneTitle: String = "User Accounts"
+	let toolbarItemIcon: NSImage = NSImage(named: NSImage.userAccountsName)!
         
 	var body: some View {
-		VStack(alignment: .preferenceSectionLabel) {
-			CheckboxSectionView(isOn1: $isOn1, isOn2: $isOn2)
-			Divider()
-				.frame(width: width, height: 20.0)
-				.alignmentGuide(.preferenceSectionLabel, computeValue: { $0[.leading] + 104.0 })
-			RadioSectionView(selection: $selection1)
-			Divider()
-				.frame(width: width, height: 20.0)
-				.alignmentGuide(.preferenceSectionLabel, computeValue: { $0[.leading] + 104.0 })
-			PickerSectionView(selection: $selection2)
-		}
-		.frame(width: width, alignment: .leading)
-		.padding([.top, .bottom], 20.0)
-		.padding([.leading, .trailing], 30.0)
-	}
-}
-
-// Some example sections with various input methods
-struct RadioSectionView: View {
-	@Binding var selection: Int
-    
-	var body: some View {
-		HStack(alignment: .top) {
-			Text("Show scroll bars:")
-				.alignmentGuide(.preferenceSectionLabel, computeValue: { $0[.trailing] })
-			Picker(selection: $selection, label: EmptyView()) {
-				Text("When scrolling").tag(0)
-				Text("Always").tag(1)
+		Preferences(contentWidth: contentWidth) {
+			PreferenceSection(title: "Permissions:") {
```

Comment:
To reduce globals, I think it would be nice to nest these things under the `Preferences` namespace using typealiases.

- `Preferences.Container`
- `Preferences.Section`

Reply:
We could maybe also do this for `PreferencesPaneView` => `Preferences.PaneView`.